src/pages/HomePage/RecentTransactions.tsx

```diff
@@ -87,15 +79,20 @@ function TransactionRow({ transaction }: { transaction: HomePageTransaction }) {
           )}
         </TransactionRecipientsWrapper>
         <TransactionRecipientsWrapper>
-          <TransactionRecipientLabel>To:</TransactionRecipientLabel>
-          {transaction.outputs.map((output) =>
-            output.__typename === "CoinOutput" ? (
-              <TransactionRecipientLink to={`/address/${output.to}`}>
-                {output.to}
-              </TransactionRecipientLink>
-            ) : (
-              output.__typename
-            )
+          {transaction.outputs.map(
```

Comment:
There's got to be an easier way. Are we using fuels TS here???

Reply:
```ts
{transaction.outputs.map(
  (output) =>
    ['ChangeOutput', 'CoinOutput', 'VariableOutput', 'WithdrawalOutput'].contains(output.__typename) && (
      <>
        <TransactionRecipientLabel>To:</TransactionRecipientLabel>
        {output.__typename}
        <TransactionRecipientLink to={`/address/${output.to}`}>
          {output.to}
        </TransactionRecipientLink>
      </>
    )
)}
```